app/controllers/learning_hours_controller.rb

```diff
@@ -36,7 +36,7 @@ def update
     authorize @learning_hour
     respond_to do |format|
       if @learning_hour.update(update_learning_hours_params)
-        format.html { redirect_to volunteer_learning_hour_path, notice: "Entry was successfully updated." }
+        format.html { redirect_to volunteer_learning_hour_path(volunteer_id: current_user.id), notice: "Entry was successfully updated." }
```

Comment:
It seems that the redirect path should be either:

```
volunteer_learning_hours_path(current_user)  # Redirect to the list (index) of learning hours for the current user
```

or 

```
redirect_to volunteer_learning_hour_path(current_user, @learning_hour) # redirect to the show page for the learning hour that was just updated
```

Or does Rails automagically supply the missing `@learning_hour` argument if it is not passed?

Reply:
It currently works fine and redirects to an updated learning_hour page. I fix this to avoid confusion anyway.